Zend/zend_vm_def.h

```diff
@@ -9808,6 +9808,26 @@ ZEND_VM_HANDLER(209, ZEND_INIT_PARENT_PROPERTY_HOOK_CALL, CONST, UNUSED|NUM, NUM
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(210, ZEND_FETCH_DEFAULT_ARG, UNUSED|NUM, UNUSED)
+{
+	USE_OPLINE
+	SAVE_OPLINE();
+
+	zend_function *called_func = EX(call)->func;
+
+	parameter_reference pr;
+	pr.offset = opline->op1.num - 1;
+	pr.required = pr.offset < called_func->common.required_num_args;
```

Comment:
I don't think we actually gain anything by populating this, but I guess it doesn't hurt to be correct.